integrations/facebook-pixel/lib/index.js

```diff
@@ -359,22 +385,25 @@ FacebookPixel.prototype.orderCompleted = function(track) {
   var products = track.products();
   var customProperties = this.buildPayload(track, true);
 
-  var contentIds = foldl(
-    function(acc, product) {
-      var item = new Track({ properties: product });
-      var key = item.productId() || item.id() || item.sku();
-      if (key) acc.push(key);
-      return acc;
-    },
-    [],
-    products
-  );
-
   var revenue = formatRevenue(track.revenue());
 
   // Order completed doesn't have a top-level category spec'd.
   // Let's default to the category of the first product. - @gabriel
   var contentType = this.getContentType(track, ['product']);
+  var contentIds = [];
+  var contents = [];
+  each(function(product) {
```

Comment:
Could we change this `each` to a traditional `for...loop` for the products array?